### PR TITLE
Doc for Doc

### PR DESCRIPTION
**Pull request by m3047**
This probably addresses https://github.com/yeti-platform/yeti/issues/405

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Adding a file and getting it to show up in the table of contents
+
+1. Edit `index.rst`. At the bottom is the table of contents directive, `toctree`. Add your filename, minus the `.rst` extension.
+1. `make clean`, then `make html`
+
+## Build Failures
+
+### `sphinxcontrib.autohttp.flask`
+
+If `sphinx-build` fails looking for this module you need to install `sphinxcontrib-httpdomain`.
```
Comment (tomchop) [inline doc/README.md]: A quick note on how to install this extension would be great. Is it just a matter of `pip install`?

File: doc/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## `make`
+
+### If you've edited a file
+
+```
+   cd doc
+   make html
+```
+`make` in turn calls `sphinx-build`.
```
Comment (tomchop) [inline doc/README.md]: I'd also point to where the generated `index.html` file is located so people can easily find it.

**Comment by m3047** [comment]
@tomchop (re-)review now. Bearing in mind that the `_build/` tree is under the directory containing this `README.md`, hopefully it's self explanatory. I added an example for `pip`. It was fine in the _Docker_ image, but I was trying to build it elsewhere when I ran into not having `httpdomain`. Didn't find the actual `pip` command in my history though, it could have been another package manager.

**Comment by tomchop** [comment]
That's great, thanks!

**Pull request merged**